common/searchattribute/defs.go

```diff
@@ -35,6 +35,7 @@ import (
 const (
 	NamespaceID           = "NamespaceId"
 	WorkflowID            = "WorkflowId"
+	ScheduleID            = "ScheduleId"
```

Comment:
maybe a comment that this isn't a real search attribute?

Reply:
I think that comment should be contextual (i.e. where the const I used). I don't think we can guarantee all future usage of the const is for fake SAs.